### DISCUSSION TITLE
how to render page, while data was inserted directly to table row

### DISCUSSION THREAD

**Discussion by zkiroel** [requarks/wiki#3008] [Help / Questions] [answered]
hello, dear community
I try to store some data to the wiki js page
I insert data to content and render fields of current id with python script and that data appear in the table,
however, I need drill down to the page Edit mode and just save it without any changes to render that page

question is - how I can make it without 'my actions'?

ps: I try to understand maybe I can solve that task with API functions, but do not found a solution yet

thank you, have a nice day

**Comment by NGPixel** [comment, marked as answer]
You should instead use the API (see https://docs.requarks.io/dev/api)

Example mutation to update a page:
```graphql
mutation {
  pages {
    update(
      id: 12345
      content: "xyz"
      description: "xyz"
      title: "xyz"
      isPublished: true
      locale: "en"
      path: "my-page"
      tags: []
    ) {
      responseResult {
        succeeded
        errorCode
        slug
        message
      }
    }
  }
}
```
